demo/kubernetes/README.md

```diff
@@ -0,0 +1,55 @@
+# Rook on Kubernetes
+
+A demo of persistent block storage from rook within a [Kubernetes](http://kubernetes.io/) cluster. We will use a mysql service in Kubernetes as an example of using block storage from rook.
+
+## Prerequisites
+
+A running Kubernetes cluster with at least 2 nodes is required. Since this demo will use the rbd volume type it will need access to modprobe on the kubernetes nodes (<https://github.com/kubernetes/kubernetes/issues/23924>). There is a fork of [coreos-kubernetes](https://github.com/coreos/coreos-kubernetes) that has a fix for this <https://github.com/rook/coreos-kubernetes/tree/rook-demo> (rook-demo branch).
+
+If you are using the rook-demo branch you can get a vagrant kubernetes cluster running like this (from the root of the repo):
+
+    $ cd multi-node/vagrant
+    $ vagrant up
+    $ export KUBECONFIG="${KUBECONFIG}:$(pwd)/kubeconfig"
+    $ kubectl config use-context vagrant-multi
+
+## Starting rook
+
+    $ kubectl create configmap rookd --from-literal=discovery-token=$(curl -w "\n" 'https://discovery.etcd.io/new?size=2')
```

Comment:
I will move curl to a new line.
As per our discussion we know etcd is something we'd like to clean up and we'll take that on outside this pull request.